example_dags/example_google_bigquery_gcs_load_and_save.py

```diff
@@ -53,13 +53,28 @@ def extract_top_5_movies(input_df: pd.DataFrame):
 
     t2 = extract_top_5_movies(input_df=t1)
 
+    # [START export_example_1] skipcq: PY-W0069
```

Comment:
Is there a way for us to fix `skipcq: PY-W0069` once? or per file? instead of having to add it on every line?

Reply:
Yup seems like adding it on top of file ignores it. Added change in [commit](https://github.com/astronomer/astro-sdk/pull/633/commits/424d86f03dfb4df0bb96f7a085510d7e220a93bc).

Reference: https://discuss.deepsource.io/t/improvement-allow-specifying-ignore-rules-in-config-file/250/2

Reply:
Okay, didn't know exactly what this `PY-W0069` was for. If it allows now adding [any commented code](https://deepsource.io/directory/analyzers/python/issues/PY-W0069) in the example I am not sure anymore if we want that.

I find it a bit weird that it thinks that this is "code" in the comment :/

Reply:
What do others think?

Reply:
I have disabled this check in deepsource -- you can now remove this from everywhere :)

Reply:
Thanks @kaxil